source/interface/dav.js

```diff
@@ -59,7 +59,7 @@ function propsToStat(props, filename, isDetailed = false) {
         type: type,
         etag: etag ? etag.replace(/"/g, "") : null
     };
-    if (type === "file") {
+    if (type === "file" && typeof mimeType === 'string') {
```

Comment:
This `typeof` would be better added to the next line so that if the mime type is _not_ a string, it returns the empty value from the ternary if.

Oh and please use double quotes :)

Reply:
just updated it accordingly